build/bin/install_tarball

```diff
@@ -42,13 +42,13 @@ ${INSTALL_COMMAND} -m 0755 queries-postgres-uptime.yml "${PMM_DIR}"/collectors/c
 ${INSTALL_COMMAND} -m 0755 queries.yaml "${PMM_DIR}"/collectors/custom-queries/postgresql/medium-resolution
 ${INSTALL_COMMAND} -m 0755 queries-mysqld-group-replication.yml "${PMM_DIR}"/collectors/custom-queries/mysql/high-resolution
 
-find ./bin -maxdepth 1 -type f | while read -r FILE; do
+for FILE in $( ls ${PWD}/bin ); do
```

Comment:
This is not clean code and will fail lint test.
Also, IMO this is the wrong way to address the problem. The problem is that the base image for the container doesn't have find, but that is trivial to fix by using a different base image and copy files in with `COPY - - from`